challenges/objects-arrays.js

```diff
@@ -43,23 +64,43 @@ const graduates = [{"id":1,"first_name":"Cynde","university":"Missouri Southern
 {"id":10,"first_name":"Hube","university":"Universitat Rovira I Virgili Tarragona","email":"hlethbrig9@foxnews.com"}]
 
 /* Request 1: Create a new array called universities that contains all the univeristies in the graduates array.  
-
 Once you have the new array created, sort the universities alphabetically and log the result. */
 const universities = [];
-console.log(universities)
+for (i = 0; i < graduates.length; i++) {
+  universities.push(graduates[i].university);
+}
+universities.sort();
+console.log(universities);
 
 /* Request 2: Create a new array called contactInfo that contains both first name and email of each student. 
-
 The resulting contact information should have a space between the first name and the email information like this: 
+
 Name email@example.com
 
 Log the result of your new array. */
 const contactInfo = [];
+
+for (i = 0; i < graduates.length; i++) {
```

Comment:
This looks great, but the instructions are looking for a different format. The output should look like: `Name email@example.com`, rather than `{
      name: graduates[i].first_name,
      email: graduates[i].email
    }`